plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -16,7 +16,8 @@ import UnitHealthUtil from "#SRC/js/utils/UnitHealthUtil";
 const COLOR_CLASSNAMES = {
   cpus: "color-1",
   mem: "color-2",
-  disk: "color-3"
+  disk: "color-3",
+  gpus: "color-1"
 };
```

Comment:
@bstavroulakis I believe the intent here was to align the colors so each resource is the same color and they would be green/yellow/red depending on the percentage. Looking at this code I'm guessing that is not the case.

cc @lilysamimi

Reply:
Hm, this seems not to be the case, the code assigns each resource with a fixed color value